checker/tests/src/tests/LockTest.java

```diff
@@ -13,7 +13,10 @@ public LockTest(File testFile) {
         super(testFile,
                 org.checkerframework.checker.lock.LockChecker.class,
                 "lock",
-                "-Anomsgtext");
+                "-AprintVerboseGenerics",
+                "-AprintAllQualifiers",
```

Comment:
printVerboseGenerics and printAllQualifiers have not effect because nomsgtext is supplied, too.